.github/workflows/ci.yml

```diff
@@ -99,151 +99,22 @@ jobs:
           ./admin/local/cli/qserv --log-level DEBUG down \
           -v
 
+      - name: Notify Slack if CI failed on main
+        uses: voxmedia/github-action-slack-notify-build@v1
+        if: github.ref == 'refs/heads/main' && failure()
+        with:
+          channel_id: G2JPZ3GC8  # this is the channel id of the dm_db_team room
+          status: FAILED
+          color: danger
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.QSERV_GHA_BUILD_NOTIFICATIONS }}
 
-
```

Comment:
How about moving this to `ci.yml.old` for the time being?  We will want to draw from this for the operator+kind CI job as soon as we bring the operator into the qserv repo